packages/amplify-graphql-default-value-transformer/src/__tests__/amplify-grapphql-default-value-transformer.test.ts

```diff
@@ -338,4 +198,90 @@ describe('DefaultValueModelTransformer:', () => {
     expect(out.resolvers['Mutation.createNote.init.1.req.vtl']).toBeDefined();
     expect(out.resolvers['Mutation.createNote.init.2.req.vtl']).toBeUndefined();
   });
+
+  it.each([{ strategy: mockSqlDataSourceStrategy() }])(
```

Comment:
Since it's important that this is a mysql data source, go ahead and specify the `dbType` argument for `mockSqlDataSourceStrategy`, so it's 1. clear that mysql is important; and 2. future-proof against default changes. (Which we hopefully won't do, but you never know...)

Reply:
Those points make a lot of sense. Added the argument.